connect/runtime/src/main/java/org/apache/kafka/connect/util/ReflectionsUtil.java

```diff
@@ -42,6 +43,14 @@ public static void registerUrlTypes() {
         Vfs.setDefaultURLTypes(urlTypes);
     }
 
+    public static URL stringtoURL(String path) {
+        try{
```

Comment:
This is a checkstyle failure (use `./gradlew testConnect` to run all those tests and checkstyle without having to run the entire, very long test suite).